utils/chain-ops/genesis/testdata/test-deploy-config-devnet-l1.json

```diff
@@ -38,6 +38,13 @@
   "colosseumSegmentsLengths": "3,11",
   "securityCouncilNumConfirmationRequired": 1,
   "securityCouncilOwners": [
-    "0x007D7e4391DCFdE47Dd7FD0B8E16091C5e0E1C7f","0x2A06Af6a4F325B1e0095a8AbD32888e0AbdCD04D","0xed92E7C40348A552822847384D722A6adB9AFeFA"
-  ]
+    "0x007D7e4391DCFdE47Dd7FD0B8E16091C5e0E1C7f",
+    "0x2A06Af6a4F325B1e0095a8AbD32888e0AbdCD04D",
+    "0xed92E7C40348A552822847384D722A6adB9AFeFA"
+  ],
+  "votingDelayBlocks": 0,
+  "votingPeriodBlocks": 25,
+  "proposalThresholdTokens": 1,
+  "votesQuorumFractionPercent": 51,
+  "proposalExecutingLatencySeconds": 1
```

Comment:
Do we need these variables in this file?

Reply:
Yes, the contracts are covered by the forge test. I'll remove this part since it's not used.